docs/projects/ActivityDekho/Activity-Dekho-DirectoryStructure.md

```diff
@@ -0,0 +1,33 @@
+
+
+# Activity Dekho
+
+AICTE Dairy - To Help Organize and Maintain your Points
+
+Visit Activity-Dekho(https://github.com/tcet-opensource/Activity-Dekho)
+
+The following is the TCET's Activity-Dekho FOLDER STRUCTURE.
+
+
+``` bash
+
+├── public
+│   └── All static files
+│   
+└── src
+    ├── components
+    │   └── All reusable and abstracted components are here
+    │   
+    ├── firebase
```

Comment:
little changes
firebase -> config.ts // contains all config options for firebase
firebase -> firestore -> _collections_ // contains all collection specific CRUD queries

Reply:
Changes are done. Checkout the updated file.